MapboxNavigation/PollyVoiceController.swift

```diff
@@ -111,18 +117,38 @@ public class PollyVoiceController: RouteVoiceController {
             return
         }
         
-        do {
-            let soundData = try Data(contentsOf: url as URL)
-            audioPlayer = try AVAudioPlayer(data: soundData)
-            audioPlayer?.delegate = self
+        pollyTask = URLSession.shared.dataTask(with: url as URL) { [weak self] (data, response, error) in
+            guard let strongSelf = self else { return }
+            
+            // If the task is canceled, don't speak.
+            // But if it's some sort of other error, use fallback voice.
+            if let error = error as NSError?, error.code == NSURLErrorCancelled {
+                return
+            } else if let error = error {
+                // Cannot call super in a closure
+                strongSelf.callSuperSpeak(strongSelf.fallbackText, error: error.localizedDescription)
```

Comment:
Do we need to call super’s `speak(_:error:)` specifically? Why not call `self.speak(_:error:)`?